src/uniparc/config/UniParcColumnConfiguration.tsx

```diff
@@ -58,6 +65,71 @@ export const UniParcColumnConfiguration = new Map<
   }
 >();
 
+/* NOTE:
```

Comment:
Seems like a good idea - would move to shared utils though?

Reply:
will do